presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcTableLayoutHandle.java

```diff
@@ -28,14 +30,23 @@
 {
     private final JdbcTableHandle table;
     private final TupleDomain<ColumnHandle> tupleDomain;
+    private final Optional<JdbcExpression> translatedSql;
 
     @JsonCreator
     public JdbcTableLayoutHandle(
```

Comment:
Need to update `equals` and `hashCode` methods

Reply:
Done - also added it to JdbcExpression as that would be necessary as well.